therminator2.sh

```diff
@@ -0,0 +1,44 @@
+package: Therminator2
+version: "%(tag_basename)s%(defaults_upper)s"
```

Comment:
Can you drop the `%(defaults_upper)s` part? We would like to move away from those for every recipe, and enforce them in the defaults themselves where it makes sense.

Reply:
Done!